config/routes.rb

```diff
@@ -136,6 +136,10 @@
         resources :devices, only: %i[create destroy]
       end
     end
+    namespace :users do
+      resources :settings, only: %i[update]
```

Comment:
blocking: same suggestion I made in the notification setting PR. Only the current user can update their own setting, so you can use a singular resource here as well

Let me know if that's not the case!